### ISSUE TITLE
MagePlaza menu disappeared from admin panel after update to 1.4.1

### ISSUE THREAD

**Issue by perederiydima1993** [mageplaza/module-core#23]
I use (mageplaza/module-smtp v1.1.3, mageplaza/magento-2-social-login                                                    2.5.1 ) and after updated, module-core from 1.3.13 to 1.4.1 disappeared menu icon.

**Comment by bien119** [comment]
Hi,

Somehow, your site cannot load the icon from pub folder. Please try to run the deploy command again!

- php bin/magento setup:static-content:deploy

I saw that you're using the old Smtp. Please try to update the latest version of module-smtp. You can find it with the link below.

- https://github.com/mageplaza/magento-2-smtp

**Comment by phuongtt1991** [comment]
Hi,
I am Matthew from Mageplaza developer team. I will close this issue. If you have still any problem with our SMTP extension, you can create a new ticket or contact with Mageplaza live chat team.
Thank you and have a nice day!

**Comment by OvalMedia** [comment]
I have the same issue. Since 1.4.1 no more Mageplaza Icon in the backend. All modules (blog, smtp, pdfinvoice) not accessible.

Sidenote: core, blog and smtp I installed using composer. Pdfinvoice for some reason did not install using composer. It complained about some dependency regarding the core module which I did not know what it meant. So I had to install pdfinvoice manually (sitting in app/code now). Not sure if this is an issue but I thought I should mention it.

Nonetheless it all worked fine before 1.4.1.
